Makefile

```diff
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := local
 
 ifndef KYMA_VERSION
-	KYMA_VERSION = 1.16.0-rc2
+	KYMA_VERSION = 1.16.0-rc3
```

Comment:
Please also adapt the version on .goreleaser.yml file

Reply:
Done